modules/reissuing-internal-certificates-for-central-services.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * configuration/reissue-internal-certificates.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="reissue-internal-certificates-central-services_{context}"]
+= Reissuing internal certificates for Central services
+
+Central uses a built-in server certificate for authentication when communicating with other {rh-rhacs-first} services.
+This certificate is unique to your Central installation.
+The {product-title-short} portal shows an information banner when the Central certificate is about to expire.
+
+[NOTE]
+====
+The information banner only appears 15 days before the certificate expiration date.
+====
+
+Beginning with {product-title-short} 4.3.4, the Operator automatically rotates the service transport layer security (TLS) certificates for all the Central components 6 months before they expire.
+
+[IMPORTANT]
+====
+* The automated rotation of the TLS certificates applies only to Operator-based installations. For all other installation methods, you must manually rotate the TLS certificates.
+
+* The rotation of the TLS certificates within the secrets does not automatically trigger the components to reload them. If the corresponding pods are not restarted at least every 6 months, you must manually restart the pods to load the new certificates before the old ones expire.
+
+ifeval::["{rhacs-version}" < "4.9.0"]
+* Certificate authority (CA) certificates are not updated. They are valid for 5 years.
```

Comment:
I didn't know this would work! Does it work with the local build? I'd definitely check and make sure it's showing the expected results after Pantheon gets hold of it. 🤞